schemas/JSON/manifests/v1.4.0/manifest.installer.1.4.0.json

```diff
@@ -65,7 +65,8 @@
         "wix",
         "burn",
         "pwa",
-        "portable"
+        "portable",
+        "advancedinstaller"

```

Comment:
nit: enum values use camelCase, if advanced installer is treated as 2 words, this should be "advancedInstaller"